microshift_support/microshift-getting-cluster-id.adoc

```diff
@@ -0,0 +1,11 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="microshift-getting-cluster-id"]
+= Getting your cluster ID
```

Comment:
Sorry for one additional question - should this say "Getting your cluster UID" for consistency? Because throughout the PR, "UID" is mentioned except only the heading.

Reply:
Please, do not change the title. 
The document is about "Cluster ID", which is implemented as a UID of a certain namespace.

Reply:
@ggiguash Thanks for clarifying

Reply:
Clster ID and UID is different. So the heading is correct.